controllers/articles.controllers.js

```diff
@@ -15,13 +16,25 @@ exports.getArticleById = (req, res, next) => {
 		});
 };
 
-exports.getArticles = (req, res) => {
-	fetchArticles().then((articles) => {
-		articles.forEach((article) => {
-			return delete article.body;
+exports.getArticles = (req, res, next) => {
+	const { topic } = req.query;
+	const articlesQuery = fetchArticles(topic);
+	const queries = [articlesQuery];
+	if (topic) {
+		const topicExistsQuery = checkTopicExists(topic);
+		queries.push(topicExistsQuery);
+	}
```

Comment:
honestly I think this touches on data validation which means it should be happening in the model

Reply:
refactored to move all data validation and associated utils from controllers to models: 
https://github.com/WolfieKnee/be-nc-news/pull/15